example/src/main/java/com/github/appintro/example/ui/custom/fragments/AnimatedDrawableFragment.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.appintro.example.ui.custom.fragments
+
+import android.graphics.drawable.AnimationDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.fragment.app.Fragment
+import com.github.appintro.example.R
+
+class AnimatedDrawableFragment : Fragment() {
+
+    private lateinit var myAnimation: AnimationDrawable
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? = inflater.inflate(R.layout.intro_animated_drawable, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<ImageView>(R.id.animated_image_view).apply {
+            setBackgroundResource(R.drawable.animated_drawable)
```

Comment:
How about if we use instead an `AnimatedVectorDrawable`?

Reply:
Oh, I've just seen that you suggested using AnimatedVectorDrawable in the issue, sorry.
I can change it today.

Reply:
The minimum api level is 21. Shall we then just leave it as is?

Reply:
Have you tried using `vectorDrawables.useSupportLibrary = true`
 https://developer.android.com/guide/topics/graphics/vector-drawable-resources#vector-drawables-backward-solution?

Reply:
Oh, didn't see, sry. I will try it out